### DISCUSSION TITLE
practice-exam/practice-exam-16

### DISCUSSION THREAD

**Discussion by giscus** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#83] [Comments]
# practice-exam/practice-exam-16

If you are preparing for AWS Cloud Practitioner Exam, this guide will help you with quick revision before the exam. I share the notes I used to study and pass exam.

https://kananinirav.com/practice-exam/practice-exam-16.html

**Comment by 4-Fingers** [comment]
I think the answer for q9 should be B & D not E (As per description provided)

**Reply by kananinirav** [reply]
Updated. Good luck with your Cloud Practitioner studies!
